host/xtest/xtest_20000.c

```diff
@@ -427,9 +427,10 @@ static void storage_corrupt(ADBG_Case_t *c,
 	ADBG_EXPECT(c, TEE_SUCCESS,
```

Comment:
This need fixing also.
I have been back through the gp specs and open session on an invalid context is an programming error.
I guess there are a full bunch of `TEE_OpenSession()` and `TEE_XXXSharedMemory()` on potentially uninited context around.
Could you have a round after them? Or we can open another pull for those (no issue reported yet).